src/RTTI/RTTIUtil.hpp

```diff
@@ -0,0 +1,144 @@
+#pragma once
+#include "RTTI_TypeIDs.hpp"
+#include <BsCorePrerequisites.h>
+#include <Private/RTTI/BsGameObjectRTTI.h>  // Says private, but bs:f uses this too in their RTTIs
+#include <Reflection/BsRTTIType.h>
+
+/**
+ * \file
```

Comment:
I was wondering, if these comment blocks describing the file (content) would be best placed at the top even above the includes. Since they are some kind of meta information, one usually wants to read these before encountering any code related stuff. What's your thought?

Reply:
I agree. Since this is a file which does not contain any class definitions, the comment should be at the top.

Reply:
Done.